### PR TITLE
Sorting and filter

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Changes made
Added sorting with
- Newest first
- Oldest first

Added filter by labels

### Preview
[Preview](https://deploy-preview-23--always-notes.netlify.app/)

### CODE REVIEW

**Review by aniketsingh13 — APPROVED** [review]
folder structure is good and also code looks good

File: src/components/RadioInput/RadioInput.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+function RadioInput({ label, name, checked, changeHandler }) {
+  return (
+    <label className="radio-input py-1">
+      <input
+        type="radio"
+        name={name}
+        checked={checked}
+        onChange={changeHandler}
+      />{" "}
```
Comment (ckahub) [inline src/components/RadioInput/RadioInput.jsx:12]: remove extra bracket

**Review by ckahub — APPROVED** [review]
code has good structure and paranthesis use
code is readable and easily understandable

File: src/components/RadioInput/RadioInput.jsx (hunk 1 above)
Comment (SanketDhabarde) [reply src/components/RadioInput/RadioInput.jsx:12]: yes

**Pull request merged**